app/src/main/java/com/example/cachua/musicdemo21/data/source/ContentResolverData.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.cachua.musicdemo21.data.source
+
+import android.content.ContentUris
+import android.content.Context
+import android.net.Uri
+import android.provider.MediaStore
+import com.example.cachua.musicdemo21.data.model.MusicModel
+
+class ContentResolverData {
```

Comment:
```suggestion
class ContentResolverData(private val context: Context) {
```
Ông có thể xem lại 2 phần này 
- Coding convention của kotlin: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules
- Constructor trong kotlin: https://kotlinlang.org/docs/reference/classes.html#constructors